dev-tools/test-patch/lucene-solr-yetus-personality.sh

```diff
@@ -113,11 +113,11 @@ function personality_modules
   case ${moduleType} in
     submodules)
       for module in "${CHANGED_MODULES[@]}"; do
-        if [[ ! "${module}" =~ ^lucene/(licenses|site) ]]; then # blacklist lucene/ dirs that aren't modules
```

Comment:
I suspect this whole file is defunct post-split, but that's another conversation

Reply:
Yes, need some cleanup in the dev-tools folder. I think we'll get to that once 9.0 work progresses...